rtos/rtx/TARGET_CORTEX_M/cmsis_os.h

```diff
@@ -69,11 +69,15 @@
 #endif
 
 // The stack space occupied is mainly dependent on the underling C standard library
+#ifdef OS_MAINSTKSIZE
+#define WORDS_STACK_SIZE OS_MAINSTKSIZE
+#else
```

Comment:
Unfortunately, this is needed. Only 32K RAM is actually available for use in UBLOX_C027. 
We must reduce main stack size and do all kind of stack reduction hacks. Otherwise we don't fit in.
Even after these drastic measures, we barely fit in.